src/data/examples/ja/09_Simulate/01_ParticleSystem.js

```diff
@@ -1,7 +1,7 @@
 /*
- * @name パーティクルシステム
- * @arialabel 点火されたスパークラーのように点から軽い灰色の円が流れ出します。
- * @description 基本的なパーティクルシステムです
+ * @name パーティクル システム
+ * @arialabel 点火されたスパークラーのように点から軽いグレーの円が流れ出します。
+ * @description 基本的なパーティクル システム
```

Comment:
[nits]

```suggestion
 * @description 基本的なパーティクルシステム
```

重箱の隅つついてすみませんが，他のファイルだとスペースはなかったので，統一したいなーと思いました！